lib/pages/chat_list/receive_sharing_intent_mixin.dart

```diff
@@ -32,7 +32,7 @@ mixin ReceiveSharingIntentMixin<T extends StatefulWidget> on State<T> {
       'file': MatrixFile(
         name: file.path,
         filePath: file.path,
-      ).detectFileType,
+      ).fromSharedMediaFile(shareFile),
```

Comment:
dont understand
create MatrixFile by `file` then get more infomation from `shareFile`
can you merge it into one function only ?

Reply:
it's factory method. don't understand what you mean ? please refer to `fromSharedMediaFile ` to understand more

Reply:
you did it wrong, let create extension for ShareMediaFile.toMatrixFile